static/versions-v1.json

```diff
@@ -3,6 +3,14 @@
     "version": "Latest",
     "id": "LATEST"
   },
+  {
+    "version": "12.9.0",
+    "id": "12.9.0"
+  },
+  {
+    "version": "12.4.117",
+    "id": "12.4.117"
+  },
```

Comment:
It is from here but seems dead link. 

https://developer.nvidia.com/cuda-toolkit-archive
```suggestion
```